backends/xnnpack/CMakeLists.txt

```diff
@@ -119,6 +118,12 @@ target_include_directories(
 target_compile_options(xnnpack_backend PUBLIC ${_common_compile_options})
 target_link_options_shared_lib(xnnpack_backend)
 
+if(EXECUTORCH_BUILD_KERNELS_OPTIMIZED)
```

Comment:
fixed build when XNNPACK is on but optimized lib is off. everything else (except corresponding change to line 149 below) is just cmake-format.